src/cli-main.ts

```diff
@@ -253,6 +254,67 @@ program
     cliUtils.logAndExit(0, copywrite);
   });
 
+program
+  .command("create-auth-token <secretKey>")
+  .description("Generates an authentication token that can be used to get detailed EdgeWorker debug response headers.  \
+  The secret key (hex-digit based) that is configured for the property in which the EdgeWorker executes.")
+  .alias("auth")
+  .option("--acl <aclPath>", "The path prefix of the response pages which require debugging; this value is used to restrict for which pages the token is valid. \
+  The default value if not specified is \"/*\". This option is mutually exclusive to the --url option; only use one or the other.")
+  .option("--url <urlPath>", "The exact path (including filename and extension) of the response page which requires debugging; this value is used as a salt for \
+  generating the token, and the URL does NOT appear in the final token itself. The generated token is only valid for the exact URL. This option is mutually \
+  exclusive to the --acl option; only use one or the other.")
+  .option("--expiry <expiry>", "The number of minutes during which the token is valid, after which it expires. Max value is 60 minutes; default value is 15 minutes.")
+  .action(async function (secretKey, options) {
+
+    if(!secretKey) {
+      cliUtils.logAndExit(1, "ERROR: The secret key specified in the property in which the EdgeWorker executes must be supplied in order to generate an authentication token.");
+    } else if(secretKey.length % 2 != 0) {
+      cliUtils.logAndExit(1, "ERROR: The secret key specified in the property in which the EdgeWorker executes must have an even number of hex characters.");
+    } else if(!secretKey.match(/^[0-9a-fA-F]+$/)) {
+      console.log("INVALID SECRET: " + secretKey);
+      cliUtils.logAndExit(1, "ERROR: The secret key specified in the property in which the EdgeWorker executes must contain only hex characters: [0-9a-f]");
+    } else {
+      secretKey = secretKey.toLowerCase();
+    }
+
+    var expiry = 15; // Use 15 minutes as the default value
```

Comment:
I dont think the if clause is necessary since you default to 15m anyway before it.

Reply:
Fixing this.